[rocprofiler] hsa-runtime64 not found

```
CMake Error at cmake_modules/env.cmake:111 (find_package):
  Could not find a package configuration file provided by "hsa-runtime64"
  (requested version 1.0) with any of the following names:

    hsa-runtime64Config.cmake
    hsa-runtime64-config.cmake

  Add the installation prefix of "hsa-runtime64" to CMAKE_PREFIX_PATH or set
  "hsa-runtime64_DIR" to a directory containing one of the above files.  If
  "hsa-runtime64" provides a separate development package or SDK, be sure it
  has been installed.
Call Stack (most recent call first):
  CMakeLists.txt:42 (include)
```

--- a/rocprofiler/.SRCINFO
+++ b/rocprofiler/.SRCINFO
@@ -1,13 +1,14 @@
 pkgbase = rocprofiler
 	pkgdesc = ROC profiler library. Profiling with perf-counters and derived metrics.
 	pkgver = 5.1.0
-	pkgrel = 1
+	pkgrel = 2
 	url = https://rocmdocs.amd.com/en/latest/ROCm_Tools/ROCm-Tools.html
 	arch = x86_64
 	license = MIT
 	makedepends = cmake
 	makedepends = python-argparse
 	makedepends = python-cppheaderparser
+	makedepends = hsa-rocr
 	depends = hip
 	depends = python
 	options = !staticlibs

--- a/rocprofiler/PKGBUILD
+++ b/rocprofiler/PKGBUILD
@@ -2,13 +2,13 @@
 # Contributor: acxz <akashpatel2008 at yahoo dot com>
 pkgname=rocprofiler
 pkgver=5.1.0
-pkgrel=1
+pkgrel=2
 pkgdesc="ROC profiler library. Profiling with perf-counters and derived metrics."
 arch=('x86_64')
 url='https://rocmdocs.amd.com/en/latest/ROCm_Tools/ROCm-Tools.html'
 license=('MIT')
 depends=('hip' 'python')
-makedepends=('cmake' 'python-argparse' 'python-cppheaderparser')
+makedepends=('cmake' 'python-argparse' 'python-cppheaderparser' 'hsa-rocr')
 options=(!staticlibs strip)
 _git='https://github.com/ROCm-Developer-Tools/rocprofiler'
 _roctracer='https://github.com/ROCm-Developer-Tools/roctracer'

